README.md

```diff
@@ -1,2 +1,160 @@
-# RoR-Budget-app
-The Ruby on Rails capstone project (remember what they are?) is about building a mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.  
+<a name="readme-top"></a>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 RoR-Budget-app <a name="about-project"></a>
+
+> The Ruby on Rails capstone project (remember what they are?) is about building a mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what. 
+
+## Built With
+
+![](https://img.shields.io/badge/-Ruby-red)
+![](https://img.shields.io/badge/-VSCode-blue)
+![](https://img.shields.io/badge/-Rspec-red)
+
+- Visual Studio Code
+- Ruby
+- Rspec
+- Rails
+
+### Key Features <a name="key-features"></a>
+
+<!-- > Describe between 1-3 key features of the application. -->
+
+- Splash screen
+- Sign up and log in pages
+- Home page (categories page)
+- Transactions page
+- "Add a new category" page
+- "Add a new transaction" page
+- Testing requirements
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link]()
+
+## Project Presentation <a name="project-presentation"></a>
+
+- [Project Presentation video link](https://www.loom.com/share/2826fc0f30414eebb472308d0c996977?sid=cfeeffa6-6c65-4d94-8d06-0e9b65a6a264)  
```

Comment:
- [ ] Thank you for adding the video and showing some of your code but some of the requirements are missing. Let's make sure you are showing all of the project features. here we can only see you showing the sign-up and sign-in parts.
![scre](https://github.com/mrkamin/RoR-Budget-app/assets/65860678/c573cbf5-8347-4981-a985-423ba758e8fd)

Reply:
@uwadonat 
thanks for your good and valuable feedback, 
as the time is very limit 5 min so I have tried to explain all part of or required in this new video presentation 
https://www.loom.com/share/21dfbd82084a431baf0c54082f2e578d?sid=2b6d7b7e-3ed3-4d1d-8fe7-f0147ba17402
I hope I could covered all things this time

Reply:
live demo = https://budgeat.onrender.com/